pkg/motor/create_bucket.go

```diff
@@ -59,5 +58,7 @@ func (mtr *motorNodeImpl) CreateBucket(ctx context.Context, request mt.CreateBuc
 
 	mtr.AddBucketServiceEndpoint(mtr.GetClient().GetRPCAddress(), cbresp.WhereIs.Did)
 
-	return b, nil
+	return &mt.CreateBucketResponse{
+		Did: cbresp.WhereIs.Did,
```

Comment:
Would it be useful to include the `WhereIs` here as well?

Reply:
Yeah imma update the response proto to include and merge @mcjcloud